engine/swarm/admin_guide.md

```diff
@@ -53,6 +53,19 @@ updates to the swarm. A quorum of managers must also agree on node additions
 and removals. Membership operations are subject to the same constraints as state
 replication.
 
+### Maintaining a quorum of managers
+
+If the swarm loses a quorum, it will not be able to perform management
```

Comment:
Not sure what's most correct here, but I've generally seen "loses quorum" instead of "loses a quorum". "Losing the quorum of managers" might also work.